pages/api/flatfile/sync-embed-records.ts

```diff
@@ -24,19 +25,22 @@ export default async function handler(
     throw new Error("No session");
   }
 
-  const { success, message } = await syncWorkbookRecords({
+  const response = await syncBenefitPlanRecords({
     workflow: WorkflowType.Embedded,
     userId: token.sub,
     organizationId: token.organizationId,
     spaceType: SpaceType.Embed,
+    actionType: ActionType.SyncEmbedRecords,
   });
-  const flash = success ? "success" : "error";
+  const flash = response?.success ? "success" : "error";
```

Comment:
Does this `response` need to be changed or did it work with the { success ,message} ?

Reply:
Reverted this change and made a change to the sync function

I was fixing the symptom not what was causing the error. Making the adjustment and merging